incuna-sass/_variables.sass

```diff
@@ -0,0 +1,20 @@
+// Set default Incuna-sass variables here
+// This way we can import incuna-sass files above
+// locally defined variables without getting errors
+
+// THESE VARIABLES CAN AND SHOULD BE OVERWRITTEN LOCALLY
+
+// Used in:
+//      forms.sass
+//      em.sass
+//      rem.sass
+$font-base: 16px !default
+
+// Used in:
+//     datepicker.sass
+//     flowplayer.sass
+//     modal.sass
+//     articles.sass
+//     dashboard.sass
+//     reminders.sass
```

Comment:
hmm alright. Just so everybody is aware @henrahmagix is telling me I can just push this change to master, so that's what I'm going to do. Henry's fault.